requirements.in

```diff
@@ -0,0 +1,9 @@
+Django==3.2.5
+django-auth-adfs==1.6.0
+django-crispy-forms==1.12.0
+djangorestframework==3.12.4
+health-check==3.4.1
+psycopg2-binary==2.9.1
+python-dateutil==2.8.1
+rac_schemas==0.26
+requests==2.25.1
```

Comment:
Looks like you don't need `requests` as a top-level dependency.

Reply:
Yep--I use it directly in a file.